builder.go

```diff
@@ -257,3 +264,11 @@ const (
 
 	defaultK6ModulePath = "go.k6.io/k6"
 )
+
+func (b Builder) osEnvOrDefaultValue(name, defaultValue string) string {
+	s := os.Getenv(name)
+	if s == "" {
```

Comment:
```suggestion
	s, ok := os.LookupEnv(name)
	if !ok {
```

This way you can set it to empty and it will just remove the flags.